### PR TITLE
Add python-wasmer

### PR DESCRIPTION
**Pull request by atrawog**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/python-wasmer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    version: {{ version }}
+    script: build_wasmer.sh       # [unix]
+
+    requirements:
+      build:
+      - {{ compiler('c') }}
+      - {{ compiler('cxx') }}
+      - {{ compiler('rust') }}
+      - cargo-bundle-licenses
+      skip: true  # [win]
```
Comment (wolfv) [inline recipes/python-wasmer/meta.yaml:115]: wrong place again :)

File: recipes/python-wasmer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+source:
+  - url: https://github.com/wasmerio/wasmer-python/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 220af5b2675480daea4ebc1cede45728ce1b270cdc141ebff08ef0870cc23cfd
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - posix  # [win]
```
Comment (wolfv) [inline recipes/python-wasmer/meta.yaml:16]: Does this do anything useful?
I think you can remove the entire section

File: .ci_support/linux64.yaml [hunk 3]
```diff
@@ -9,6 +9,6 @@ target_platform:
 channel_sources:
 - conda-forge
 docker_image:
-- quay.io/condaforge/linux-anvil-comp7
+- quay.io/condaforge/linux-anvil-cos7-x86_64
```
Comment (wolfv) [inline .ci_support/linux64.yaml:12]: can you undo these changes?

COS7 will be automatically requested I believe when you add teh right things as requirements in the recipe.

File: recipes/python-wasmer/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    script: build_wasmer.bat      # [win]
+    build:
+      run_exports:
+        - python-wasmer
+    requirements:
+      build:
+      - {{ compiler('c') }}
+      - {{ compiler('cxx') }}
+      - {{ compiler('rust') }}
+      - sysroot_linux-64 2.17  # [linux64]
```
Comment (wolfv) [inline recipes/python-wasmer/meta.yaml:31]: I think this line automatically requests COS7 docker image so you don't need to modify the otehr files.

File: .ci_support/linux64.yaml (hunk 3 above)
Comment (atrawog) [reply .ci_support/linux64.yaml:12]: I tried, but ```run_docker_build.sh``` is using ```DOCKER_IMAGE=$(cat .ci_support/${CONFIG}.yaml | grep '^docker_image:$' -A 1 | tail -n 1 | cut -b 3-)``` to pick the image and doesn't check if you have a ```sysroot_linux-64 2.17``` build requirement or not.

File: recipes/python-wasmer/meta.yaml (hunk 4 above)
Comment (ngam) [reply recipes/python-wasmer/meta.yaml:27]: To be safe, you could include cos7 in conda-forge.yml like this:

https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7

```yaml
os_version:
  linux_64: cos7
```

File: .ci_support/linux64.yaml (hunk 3 above)
Comment (ngam) [reply .ci_support/linux64.yaml:12]: You should definitely not have this here like wolfv suggested. Try adding a conda-forge.yml file next to meta.yaml with

```
os_version:
  linux_64: cos7
```

That should almost certainly do the trick.

**Review by ngam — APPROVED** [review]
@wolfv or @conda-forge/staged-recipes this seems ready to go
